### PR TITLE
Remove partest orphan files

### PR DESCRIPTION
**Pull request by ashawley**
There are still extra files after #7255.  

These shell commands help to find them.

```
$ find . -type f -name '*.flags'
         ! -exec sh -c 'test -f ${0%.flags}.scala -o -d ${0%.flags}/' {} \;
```

```
$ find . -type f -name '*.check'
         ! -exec sh -c 'test -f ${0%.check}.scala -o -d ${0%.check}/' {} \;
```

I wonder if they should be used as lint rules for partest suites, or could be added to partest itself.

### CODE REVIEW

File: test/files/instrumented/indy-symbol-literal.check [hunk 1]
```diff
@@ -1,6 +0,0 @@
-indy-symbol-literal.scala:10: warning: a pure expression does nothing in statement position; multiline expressions might require enclosing parentheses
-      'foo.name
-           ^
-indy-symbol-literal.scala:6: warning: a pure expression does nothing in statement position; multiline expressions might require enclosing parentheses
-    'warmup
-    ^
```
Comment (ashawley) [inline test/files/instrumented/indy-symbol-literal.check:6]: Was this an accidentally added file in #5310, or was there a test to go along with it that was missed?

Comment (som-snytt) [reply test/files/instrumented/indy-symbol-literal.check:6]: History of `instrumented` shows the test was deleted by https://github.com/scala/scala/pull/6864

**Comment by som-snytt** [comment]
https://github.com/scala/scala/pull/7301 introduces a red flag.

```
partest --instrumented
[snip]
Extraneous file: /Users/[snip]/scala/test/files/instrumented/indy-symbol-literal.check
Selected 3 tests drawn from 1 named test categories

# starting 3 tests in instrumented
```

**Review by som-snytt — APPROVED** [review]

File: test/files/instrumented/indy-symbol-literal.check (hunk 1 above)
Comment (ashawley) [reply test/files/instrumented/indy-symbol-literal.check:6]: Ok, I'll keep it deleted, then.

**Comment by SethTisue** [comment]
thank you Aaron!

**Pull request merged**